can-setup.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+usage() {
+    cat <<EOF
+$0 [--config|-c <file>] [--iface|-i <interface>] [start|up] [stop|down]
+
+If no parameters are given, then "zcan" network interface and "zcan.conf"
+configuration file are used. The script waits until user presses CTRL-c
+and then removes the canbus network interface.
```

Comment:
The default (today) is ``zcan0``. I suggest we update the usage and defaults to use that.